drf_spectacular/settings.py

```diff
@@ -112,6 +112,8 @@
     'OAUTH2_TOKEN_URL': None,
     'OAUTH2_REFRESH_URL': None,
     'OAUTH2_SCOPES': None,
+    # Option for turning off error and warn messages
+    'DISABLE_ERRORS_AND_WARNINGS_MESSAGES': False,
```

Comment:
could you name it `DISABLE_ERRORS_AND_WARNINGS` and move it to line 85 (above the static schema additions) with empty lines around it? thanks.